scissors/src/main/java/com/lyft/android/scissors/GlideBitmapLoader.java

```diff
@@ -42,21 +43,22 @@ public GlideBitmapLoader(@NonNull RequestManager requestManager, @NonNull Bitmap
 
     @Override
     public void load(@Nullable Object model, @NonNull ImageView imageView) {
-        requestManager.load(model)
-                .asBitmap()
-                .skipMemoryCache(true)
-                .diskCacheStrategy(DiskCacheStrategy.SOURCE)
-                .transform(transformation)
-                .into(imageView);
+        RequestOptions requestOptions = new RequestOptions();
+        requestOptions.skipMemoryCache(true)
+                .diskCacheStrategy(DiskCacheStrategy.DATA)
+                .transform(transformation);
+
+        requestManager.asBitmap()
+                .load(model)
+                .apply(requestOptions);
```

Comment:
These were the simplest changes. Glide 4 now has this `RequestOptions` object that receives some calls we used to make directly on our `RequestBuilder`.

Also, `DiskCacheStrategy. SOURCE ` translates to the new `DiskCacheStrategy.DATA`. There's a [nice blog post](https://futurestud.io/tutorials/glide-caching-basics) with a summary about it. But here's the documentation just in case:

[The old `SOURCE`](https://github.com/bumptech/glide/blob/3.0/library/src/main/java/com/bumptech/glide/load/engine/DiskCacheStrategy.java#L12):
>Saves just the original data to cache.

[The new `DATA`](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/engine/DiskCacheStrategy.java#L67):
>Writes retrieved data directly to the disk cache before it's decoded.

Reply:
Appreciate the details:-)